packages/confirm-dialog/src/vaadin-confirm-dialog.js

```diff
@@ -68,7 +69,9 @@ import { ConfirmDialogMixin } from './vaadin-confirm-dialog-mixin.js';
  * @mixes ElementMixin
  * @mixes ThemePropertyMixin
  */
-class ConfirmDialog extends ConfirmDialogMixin(ElementMixin(ThemePropertyMixin(PolylitMixin(LitElement)))) {
+class ConfirmDialog extends ConfirmDialogMixin(
```

Comment:
BTW we don't need `CSSInjectionMixin` for `vaadin-confirm-dialog` too as it's not supposed to be stylable.
This can be probably changed with native popovers refactor so maybe it's ok to keep it here for now.

Reply:
Removed.